Foundation.md

```diff
@@ -0,0 +1,34 @@
+# Foundation
+
+[Foundation](http://foundation.zurb.com/) is the one of the most advanced responsive front-end frameworks.
+
+## Getting Started
+
+Here is a simple HTML template which includes the latest compiled and minified CSS and Javascript for the Foundation library.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Foundation Template</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/foundation/6.2.1/foundation.min.css">
+  </head>
+  <body>
+    <h1>Hello World</h1>
+    <!-- Add all HTML Code here -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/foundation/6.2.1/foundation.min.js"></script>
+  </body>
+</html>
+```
+
+We have used a CDN in this example, but you can checkout other ways of installing Foundation [here](http://foundation.zurb.com/sites/download/).
+
+## Learning Resources
+
+- The official documentation for Foundation is available [here](http://foundation.zurb.com/sites/docs/).
+- Checkout Foundation's open source GitHub repository [here](https://github.com/zurb/foundation-sites).
+- [Here](http://foundation.zurb.com/learn/tutorials.html) is a list of tutorials on using the Foundation Framework.
```

Comment:
- :point_right: List of [**`Foundation Framework tutorials`**](http://foundation.zurb.com/learn/tutorials.html)